setup.py

```diff
@@ -10,7 +10,7 @@
 
 import versioneer
 
-description = ("A template QIIME 2 plugin.")
+description = "A QIIME 2 plugin for viromics identification and analysis."
```

Comment:
```suggestion
description = "A QIIME 2 plugin for the identification and analysis of viral sequences."
```
identification of viromics does not make sense. Mine is a suggestion maybe you find something better.